### PR TITLE
Remove duplicate NC name on neighborhood recent dashboard

### LINKED ISSUE #1230: Fix Incorrect Data on 311 Data Dashboard
### Overview
The Neighborhood Weekly Report on the 311 Data Dashboard shows an incorrect city name.

### Action Items
- [ ] Visit https://dev.311-data.org/map and navigate (through the top menu) to Reports-> Neighborhood Weekly
- [ ] See that first line of text below the drop-down menu says 'Arleta weekly report (dates)' remains the same as you change to different cities in the drop-down menu.
- [ ] Fix this bug by having the city name in the line of text match the item selected in the drop-down menu.
   - See https://dev.311-data.org/map Reports -> Neighborhood for an example where the title of the charts reflect the city selected in the drop-down menu.

### Resources/Instructions
[311 Data Dashboard](https://dev.311-data.org/map)

neighborhood_mismatch

### PR DESCRIPTION
**Pull request by joshuayhwu**
Fixes #1230

- Resolve issue with inconsistent NC dropdown and description name, but maintains correct filtering mechanism.  
 
  - [ ] All PR Status checks are successful
  - [ ] Peer reviewed and approved

Any questions? See the [getting started guide](https://github.com/hackforla/311-data/blob/master/GETTING_STARTED.md)

Potentially a additional end-of-line when saving files using vscode, see [here](https://gist.github.com/camh-/1bebfcff1b0f814e9b191edc60d5206b)

### CODE REVIEW

File: server/api/alembic/seeds/types.csv [hunk 1]
```diff
@@ -8,6 +8,6 @@ type_id,type_name,agency_id,color,description,data_code
 7,Metal/Appliances,2,#8B508B,"Air conditioners, dryers, refrigerator and more...",Metal/Household Appliances
 8,Single Streetlight,1,#79B74E,"Pole knocked down, streetlight outage on a wooden power pole, or malfunctioning traffic signal",Single Streetlight Issue
 9,Multiple Streetlights,1,#EDAD08,"Multiple poles knocked down, streetlight outages on wooden power poles, or malfunctioning traffic signals",Multiple Streetlight Issue
-10,Water Waste,1,#EDAD08,"Water runoff, over-watering, incorrect water days, or any other water waste ",Report Water Waste
+10,Water Waste,1,#DCED08,"Water runoff, over-watering, incorrect water days, or any other water waste ",Report Water Waste
```
Comment (nichhk) [inline server/api/alembic/seeds/types.csv:11]: Revert this change since it's part of #1226

File: server/dash/design.py [hunk 2]
```diff
@@ -60,7 +59,7 @@
     "Metal/Appliances": "#8B508B",
     "Single Streetlight": "#79B74E",
     "Multiple Streetlights": "#EDAD08",
-    "Water Waste": "#EDAD08",
+    "Water Waste": "#DCED08",
```
Comment (nichhk) [inline server/dash/design.py:62]: ditto

File: server/dash/design.py [hunk 3]
```diff
@@ -47,7 +47,6 @@
 }
 
 
-# TODO: Fixing multiple streetlights and waterwaste color
```
Comment (nichhk) [inline server/dash/design.py:50]: ditto

File: server/dash/dashboards/neighborhood_recent.py [hunk 4]
```diff
@@ -174,6 +174,7 @@ def update_table(selected_council):
     Input("council_list", "value")
 )
 def update_text(selected_council):
+    print(selected_council)
```
Comment (nichhk) [inline server/dash/dashboards/neighborhood_recent.py:177]: If we want to log this in prod, let's try to make this as descriptive as possible, e.g.,
`print("neighborhood_recent.update_text: selected_council is ", selected_council)`

**Review by nichhk — APPROVED** [review]
Thanks Josh! A few comments, but looks good to submit afterwards.

File: server/dash/dashboards/neighborhood_recent.py (hunk 4 above)
Comment (joshuayhwu) [reply server/dash/dashboards/neighborhood_recent.py]: Thanks for the catch - removing print statement for testing purposes.

File: server/dash/design.py (hunk 3 above)
Comment (joshuayhwu) [reply server/dash/design.py]: Thanks for the catch - reverted change

File: server/api/alembic/seeds/types.csv (hunk 1 above)
Comment (joshuayhwu) [reply server/api/alembic/seeds/types.csv]: Resolve

File: server/dash/design.py (hunk 2 above)
Comment (joshuayhwu) [reply server/dash/design.py]: Thanks for the catch - resolve
